src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_integration_test.py

```diff
@@ -662,3 +664,119 @@ def test_merge_reductions(self, mock_get_timeout):
     """Tests merging. Wrapper around _test_merge_reductions."""
     mock_get_timeout.return_value = get_fuzz_timeout(1.0)
     self._test_merge_reductions('minijail-merge')
+
+
+@test_utils.integration
+@test_utils.with_cloud_emulators('datastore')
+class TestLauncherZFuchsia(BaseLauncherTest):
+  """libFuzzer launcher tests (Fuchsia)."""
+
+  def setUp(self):
+    # Set up a Fuzzer.
+    data_types.Fuzzer(
+        revision=1,
+        additional_environment_string=
+        'FUCHSIA_RESOURCES_URL = gs://fuchsia-on-clusterfuzz-v2/*',
+        builtin=True,
+        differential=False,
+        file_size='builtin',
+        jobs=[u'libfuzzer_asan_test_fuzzer'],
```

Comment:
Do you need to make this string a unicode string? If not please remove the u (same for other strings in this CL)